Panel content reorganization + Control/Tool rework (#5384)
scalableminds/webknossos: docs/getting_started.md

## Your First Annotation
Click the `Create Annotation` button from the view mode of a dataset to create your first annotation.
webKnossos will launch the main annotation screen allowing you to navigate your dataset, place markers to reconstruct skeletons, or annotate segments as volume annotations.

> Depending on the current tool which can be selected in the top bar, various actions can be performed.

Reviewer: ```suggestion
Depending on the current tool - selectable in the top bar - various actions can be performed.
```

Note that the most important controls are always shown in the status bar at the bottom of your screen.

> The first tool is the move tool which allows to navigate the dataset by moving the mouse while holding the left mouse button.

Reviewer: ```suggestion
The first tool is the MOVE tool which allows to navigate the dataset by moving the mouse while holding the left mouse button.
```

> In the skeleton tool, left click can be used to place markers in the data, called nodes.

Reviewer: ```suggestion
In the SKELETON tool, left click can be used to place markers in the data, called nodes.
```

Additionally, the left mouse button can also be used to navigate around, select or drag nodes.

> The draw tool allows to "paint" voxels to create volumetric annotations.

Reviewer: ```suggestion
The DRAW tool allows to "paint" voxels to create volumetric annotations.
```
